src/inverseRobot.js

```diff
@@ -23,8 +23,25 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const reversedObj = {};
+  const properties = Object.keys(robot);
+  const values = Object.values(robot);
+
+  for (let i = 0; i < properties.length; i++) {
+    reversedObj[values[i]] = properties[i];
+  }
+
+  const sortedValues = values.sort();
```

Comment:
You definitely don't need this. Use nested loops:
for(const properties in robot) {
      for (const reversedProp in reversedObj) {
        // do the magic here
  }
  // do another magic here
}

Reply:
thanks for giving a hint about a better solution of this task.